crates/forge/tests/data/trace/tests/test_trace.cairo

```diff
@@ -0,0 +1,13 @@
+use snforge_std::{declare, ContractClassTrait, PrintTrait};
+use snforge_std::trace::{get_last_call_trace, DisplayArrayCallEntryPoint};
```

Comment:
This trait impl import is needed bcs sth in cairo is not right when it comes to some Display imports and print macros, see https://github.com/starkware-libs/cairo/pull/4745#pullrequestreview-1805963713